pixels/pixels.py

```diff
@@ -42,6 +43,13 @@
     docs_url=None,
     redoc_url=None,
 )
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_methods=["GET", "HEAD"],
+    allow_headers=["*"],
+    allow_credentials=True,
```

Comment:
https://fastapi.tiangolo.com/tutorial/cors/ says that "Also, `allow_origins` cannot be set to ['*'] for credentials to be allowed, origins must be specified.".

Does that mean we should remove this line? We don't use cookies in the app for anything other than the auth flow anyway, so no need in enabling this.

Reply:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials):
> Credentials are cookies, **authorization headers** or TLS client certificates.
Emphasis mine. But yeah it looks like we can't set wildcard origins and allow credentials, so idk.

Reply:
Actually I think that might just refer to not automatically forwarding an `Authorization` header the client used to get the page it is making the request from, but I'm not sure.

Reply:
Hmm, should we omit this flag for now then, merge this PR and see what happens when people implement cross-origin services?

Reply:
Sure